packages/node_modules/@webex/plugin-meetings/src/meetings/index.js

```diff
@@ -606,15 +612,27 @@ export default class Meetings extends WebexPlugin {
      * @memberof Meetings
      */
     syncMeetings() {
-      return this.request.getActiveMeetings().then((locus) => {
-        if (locus.loci && locus.loci.length > 0) {
-          locus.loci.forEach((locus) => {
+      return this.request.getActiveMeetings().then((locusArray) => {
+        const activeLocusUrl = [];
+
+        if (locusArray.loci && locusArray.loci.length > 0) {
+          locusArray.loci.forEach((locus) => {
+            activeLocusUrl.push(locus.url);
             this.handleLocusEvent({
               locus,
               locusUrl: locus.url
             });
           });
         }
+        else if (Object.keys(this.meetingCollection.getAll()).length > 0) {
+          // Some time the mercury event is missed after mercury reconnect
+          // if sync returns no locus then clear all the meetings
```

Comment:
Do you mean the event is missed while we are disconnected?

Reply:
yep i tried it manually